app/assets/javascripts/dashboard/views/dash/advanced_dataset/advanced_dataset_view.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+/* eslint-disable jsx-a11y/href-no-hash, react/prefer-stateless-function */
+
+import React from "react";
+import TemplateHelpers from "libs/template_helpers";
+import type { APIUserType, APIDatasetType } from "admin/api_flow_types";
+import Utils from "libs/utils";
+import { Table, Icon } from "antd";
+import DatasetActionView from "./dataset_action_view";
+import DatasetAccessListView from "./dataset_access_list_view";
+
+const { Column } = Table;
+
+type Props = {
+  datasets: Array<APIDatasetType>,
+  searchQuery: string,
+};
+
+class AdvancedDatasetView extends React.PureComponent {
+  props: Props;
+
+  render() {
+    const compareFunc = (attribute: string) => (a: Object, b: Object) =>
+      a[attribute].toLowerCase().localeCompare(b[attribute].toLowerCase());
+
+    return (
+      <div>
+        <Table
+          dataSource={Utils.filterWithSearchQuery(
+            this.props.datasets,
+            ["name", "owningTeam", "description"],
+            this.props.searchQuery,
+          )}
+          rowKey="name"
+          pagination={{
+            defaultPageSize: 50,
+          }}
+          expandedRowRender={dataset => <DatasetAccessListView dataset={dataset} />}
+        >
+          <Column
+            title="Name"
+            dataIndex="name"
+            key="name"
+            sorter={compareFunc("name")}
+            render={(name, dataset: APIDatasetType) =>
+              <div title={dataset.dataSource.baseDir}>
```

Comment:
The title attribute is used for tooltip generation by the browser when hovering over the element. However, the server does not seem to send the baseDir information? Doesn't work on the master either, but maybe this is due to my local datasets. In any case, it shouldn't break anything.